src/analysis_pipeline_utils/utils_analysis_dispatch.py

```diff
@@ -52,22 +58,27 @@ def get_data_asset_paths_from_query(
                 "$group": {
                     "_id": "$" + group_by if group_by else "$_id",
                     "asset_location": {"$push": f"${asset_id_prefix}"},
+                    "docdb_id": {"$push": "$_id"},
                 }
             },
         ]
     )
 
-    return [x["asset_location"] for x in response]
+    locations = [x["asset_location"] for x in response]
+    docdb_ids = [x["docdb_id"] for x in response]
+    return locations, docdb_ids
 
 
-def get_s3_input_information(
+def get_s3_and_docdb_input_information(
     data_asset_paths: List[str],
+    docdb_record_ids: List[str],
     file_extension: str = "",
     split_files: bool = True,
-) -> tuple[List[str], Union[List[str], List[List[str]]]]:
+) -> tuple[List[str], Union[List[str], List[List[str]]], List[str]]:
     """
-    Returns tuple of list of s3 buckets, list of s3 asset ids,
-    and list of s3 paths, looking for the file extension if specified
+    Returns tuple of list of s3 buckets, list of s3 paths,
+    looking for the file extension if specified and list of
+    docdb record ids for each asset
```

Comment:
I don't think this needs docdb_record_ids - it seems to be returning them unchanged, along with data_asset_paths. Change this to return just s3_file_paths?